### ISSUE TITLE
[FR]: Magisk kitsune Rezygisk module need instead of its' zygisk!

### ISSUE THREAD

**Issue by MIkeeJY** [PerformanC/ReZygisk#58]
### Description

Magisk kitsune only can enable zygisk by itself，I need a module that can support zygisk instead of itself zygisk if enable

### Reason

Kitsune won't update like magisk alpha,so I need this help~

### Confirmations

- [X] This feature is not already implemented.
- [X] I have verified that this is not a duplicate feature request.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by devnoname120** [comment]
Why do you use Magisk Kitsune? It's crap

**Comment by MIkeeJY** [comment]
> It's crap

Yeah I know, I usually use magisk alpha,but I can only Magisk Kitsune to get root in docker image

**Comment by Kevinco1** [comment]
> ### Description
> Magisk kitsune only can enable zygisk by itself，I need a module that can support zygisk instead of itself zygisk if enable
> 
> ### Reason
> Kitsune won't update like magisk alpha,so I need this help~
> 
> <img alt="image" width="274" src="https://private-user-images.githubusercontent.com/8580762/375956606-13c7a2a4-56aa-47b3-985c-a909dc70c7ec.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjkwNTA4MzIsIm5iZiI6MTcyOTA1MDUzMiwicGF0aCI6Ii84NTgwNzYyLzM3NTk1NjYwNi0xM2M3YTJhNC01NmFhLTQ3YjMtOTg1Yy1hOTA5ZGM3MGM3ZWMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTAxNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEwMTZUMDM0ODUyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YWYwM2Q0NmNlNDFjMWU3ZWYyZWI3OGJlMjc0NmViOGU5NjE4MzRkNmFiODExM2Q3YTRlYjBiZjY2MzkwYTE4MSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.tH7PK9A6MOoMgQKoJoLfEJIgSFKLy96Tsz3Hhjvuct8"> <img alt="image" width="262" src="https://private-user-images.githubusercontent.com/8580762/375956625-27f6f3f3-5bad-4760-b3de-049cd645b083.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjkwNTA4MzIsIm5iZiI6MTcyOTA1MDUzMiwicGF0aCI6Ii84NTgwNzYyLzM3NTk1NjYyNS0yN2Y2ZjNmMy01YmFkLTQ3NjAtYjNkZS0wNDljZDY0NWIwODMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTAxNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEwMTZUMDM0ODUyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NGZmNzViNTcxM2ExYmM3OTBmNzNlNTAxZThmNDAzNjRmODViNjZlYjBmYTAyZWVlNWFkOGVlODE4ZWMyMDRiOCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.AmzGq9biX7zhqj22T0YYeTLA6x6adJSXJlqPF0kXM_g">
> ### Confirmations
> * [x]  This feature is not already implemented.
> * [x]  I have verified that this is not a duplicate feature request.
> 
> ### Code of Conduct
> * [x]  I agree to follow this project's Code of Conduct

Use ZygiskNext

**Comment by MIkeeJY** [comment]
@Kevinco1 Zygisk Next not work,bro~See my pics

**Comment by Kevinco1** [comment]
> @Kevinco1 Zygisk Next not work,bro~  Not support for kitsune~

Thanks for the info. Haven't used Kistune in a while since it hasn't been updated

**Comment by ThePedroo** [comment]
> Use ZygiskNext
I don't think that's appropriate for saying in ReZygisk repository

**Comment by MIkeeJY** [comment]
https://github.com/Dr-TSNG/ZygiskNext/releases/tag/v1.2.0  has supported~
